src/devices/Tca955x/samples/Tca955x.Sample.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Spi;
+using System.Threading;
+using Iot.Device.Tca955x;
+
+var tca9554 = GetTca9554Device();
+
+// Set the first 4 bits to Input the others as Output
```

Comment:
I guess those few first lines are when you want to directly manipulate the tca955x?
And you can use it as well as a GPIO Controller like in the second part of the sample?
Correct?
In all cases, can you be a bit more clear on if anything needs to be done before and why it's necessary? Or jus be clear that you can actually do both.